### PR TITLE
Update to use framework 

### PR DESCRIPTION
**Pull request by brammerl**
# Summary

Update the mux data source primary installation to point towards the install data source nerdlet to use the framework. 

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by brammerl** [comment]
Waiting on product partnerships to approve before merging. They will reach out to hero to merge when ready to release.

**Comment by d3caf** [comment]
@brammerl this looks to not be passing checks. Would you mind fixing it?

**Comment by d3caf** [comment]
Also @jcountsNR, I think we could use your review here!

File: data-sources/mux/config.yml
Comment (JuliaNocera) [inline data-sources/mux/config.yml:8]: Looks like we need to add a field above nerdletId and nerdletState of `nerdlet`. Here's an example https://github.com/newrelic/newrelic-quickstarts/blob/370d7c9791f26c48e876e2dadb8b7d8ff4d229f5/data-sources/unix/config.yml#L10

**Review by seemantk — APPROVED** [review]

**Review by jcountsNR — APPROVED** [review]
Looks good to me. Not sure why the FOSSA CLI is failing, must be an internal issue because it doesn't seem relevant to the PR.

**Pull request merged**
